packages/client/hmi-client/src/components/widgets/tera-infinite-canvas.vue

```diff
@@ -228,5 +228,6 @@ svg:active {
 
 .foreground-layer {
 	position: relative;
+	pointer-events: none;
 }
```

Comment:
Infinite-canvas is used in other places (e.g. workflow), as well the top two layers (foreground, data) are meant to be interactive. 

Adding a `pointer-event: none` here will break other areas of the application.